packages/calypso-e2e/src/lib/test-account.ts

```diff
@@ -56,7 +56,15 @@ export class TestAccount {
 		}
 		if ( waitUntilStable ) {
 			const sidebarComponent = new SidebarComponent( page );
-			await sidebarComponent.waitForSidebarInitialization();
+
+			// Wait for either of the WP-Admin or Calypso
+			// sidebars to appear.
+			// @TODO Replace the raw WP-Admin selector with
+			// a POM.
+			await Promise.race( [
+				sidebarComponent.waitForSidebarInitialization(),
+				page.locator( 'div[id="adminmenuwrap"]' ).waitFor( { timeout: 20 * 1000 } ),
+			] );
```

Comment:
**Key change**

The `waitUntilStable` flag is only used by a few key flows, all of them being Calypso sites. All except `ecomm-plan` user for Jetpack AT use Calypso, but we need to handle the scenario of eCommerce plan user possibly specifying this flag.